cli/tools/coverage.rs

```diff
@@ -212,41 +230,56 @@ impl PrettyCoverageReporter {
   }
 }
 
-pub fn filter_script_coverages(
-  coverages: Vec<Coverage>,
-  test_file_url: Url,
-  test_modules: Vec<Url>,
-) -> Vec<Coverage> {
-  coverages
+fn collect_coverages(
+  dir: &PathBuf,
+  _ignore: Vec<PathBuf>,
+) -> Result<Vec<Coverage>, AnyError> {
+  let mut coverages: Vec<Coverage> = Vec::new();
+
+  let entries = fs::read_dir(dir)?;
+  for entry in entries {
+    let json = fs::read_to_string(entry.unwrap().path())?;
+    let coverage: Coverage = serde_json::from_str(&json)?;
+
+    coverages.push(coverage);
+  }
+
+  coverages = coverages
     .into_iter()
-    .filter(|e| {
-      if let Ok(url) = Url::parse(&e.script_coverage.url) {
-        if url.path().ends_with("__anonymous__") {
-          return false;
-        }
+    .filter(|e| !e.script_coverage.url.ends_with("__anonymous__"))
+    .collect::<Vec<Coverage>>();
```

Comment:
**TODO** filter out anything that starts with any patterns in ignore.
Ignore should default to include files starting with the basename `test`.

Reply:
Most other coverage reporting tools (nyc, kvoc, grcov, etc) use include and exclude, where an include match would override a match in exclude.

Went with ignore because its what we use in the other commands like doc and lint but we might want to reconsider

Reply:
Reverted to a simpler filter for the time being.